packages/keystone/cache.js

```diff
@@ -29,6 +29,13 @@
 const express = require('express')
 const { get, set } = require('lodash')
 
+const ENABLE_CACHE_LOGGING = true
```

Comment:
Why not from .env? Is it WIP? When why not tagged?

Reply:
yep, this is wip. Forgot to tag. Gonna dismiss and re-request your review once ready